<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserimentoutente_window</class>
 <widget class="QMainWindow" name="inserimentoutente_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>308</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>308</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>308</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inserimento Utente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xz.qrc">
    <normaloff>:/images/images/AnconAmbiente_Logo.png</normaloff>:/images/images/AnconAmbiente_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="inserimentoutente_widget">
   <widget class="QPushButton" name="salva_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>SALVA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>161</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Operatori:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PERMESSI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annulla_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>ANNULLA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>191</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Servizi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>221</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Clienti:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="operatori_check">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="servizi_check">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="clienti_check">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Turni:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="utenti_check">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>219</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Utenti:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mezzi_check">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>159</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="turni_check">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>189</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mezzi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_error">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>230</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="password_error">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>230</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xz.qrc"/>
 </resources>
 <connections/>
</ui>
